src/environment.ts

```diff
@@ -69,7 +70,7 @@ export class SecureEnvironment implements MembraneBroker {
             throw ErrorCreate(`Missing SecureEnvironmentOptions options bag.`);
         }
         const { rawGlobalThis, secureGlobalThis, distortionMap } = options;
-        this.distortionMap = WeakMapCreate(isUndefined(distortionMap) ? [] : distortionMap.entries());
+        this.distortionMap = WeakMapCreate(isUndefined(distortionMap) ? [] : MapEntries(distortionMap));
```

Comment:
@dejang we discussed this one in the pass, might we should just add a comment here, but the options is an object provided by the user, which we require to comply with the `Map` api, but it could be a subclass, etc., for that reason, we should not force to do a control extraction of the entries.

Reply:
I understand. Makes sense. I'll close this PR since there's no need for it.